sdk/python/packages/flet-core/src/flet_core/flashlight.py

```diff
@@ -18,7 +18,8 @@ def main(page: ft.Page):
         flashLight = ft.Flashlight()
         page.overlay.append(flashLight)
         page.add(
-            ft.TextButton("On", on_click:lambda _: flashLight.on()),
+            ft.TextButton("On", on_click:lambda _: flashLight.turn_on()),
+            ft.TextButton("Off", on_click:lambda _: flashLight.turn_off()),
             ft.TextButton("Off", on_click:lambda _: flashLight.off()),
```

Comment:
Please delete this line (i requested changes, but i didn't know that it's will add a new line with `turn_off()` method

Reply:
i am on vacation, and I'm gonna do the editing after i return.

Reply:
Ok.

Reply:
I got a question. 

Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?

Reply:
> @bleudev
> 
> I am back from the vacation.
> 
> After reading your suggestion, I got a question.
> 
> Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?

I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?

Reply:
> > @bleudev
> > I am back from the vacation.
> > After reading your suggestion, I got a question.
> > Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?
> 
> I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?

Maybe you want to use only `toggle()` without `turn_on()` and `turn_off()`?

Reply:
> > @bleudev
> > I am back from the vacation.
> > After reading your suggestion, I got a question.
> > Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?
> 
> I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?


I was asking if `turn_on()` and `turn_off()` should be documented too or if just documenting `toggle()` is enough...

Reply:
> > > @bleudev
> > > I am back from the vacation.
> > > After reading your suggestion, I got a question.
> > > Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?
> > 
> > 
> > I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?
> 
> I was asking if `turn_on()` and `turn_off()` should be documented too or if just documenting `toggle()` is enough...

Please sorry, but i don't understand well right now. Can you send two variants of code for understanding your question? For example one where you documented only `toggle()` and another one where you documented all methods.

Please sorry i may be stupid in not understanding your question :)
You can also do whatever you want.

Reply:
> > > > @bleudev
> > > > I am back from the vacation.
> > > > After reading your suggestion, I got a question.
> > > > Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?
> > > 
> > > 
> > > I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?
> > 
> > 
> > I was asking if `turn_on()` and `turn_off()` should be documented too or if just documenting `toggle()` is enough...
> 
> Please sorry, but i don't understand well right now. Can you send two variants of code for understanding your question? For example one where you documented only `toggle()` and another one where you documented all methods.
> 
> Please sorry i may be stupid in not understanding your question :) You can also do whatever you want.

My bad actually 😅,

By documenting i mean when making a pull request for the website, should i say the functions `turn_on()` and `turn_off` exist or not.

Sorry for not being easy to understand

Reply:
> > > > > @bleudev
> > > > > I am back from the vacation.
> > > > > After reading your suggestion, I got a question.
> > > > > Do you think that `toggle` is enough or should `turn_on` and `turn_off` be exposed too?
> > > > 
> > > > 
> > > > I don't understand your question :) Sorry, i use translator, because i don't speak English very well. I think that this docstring after last commit is fixed. Can you explain your question better?
> > > 
> > > 
> > > I was asking if `turn_on()` and `turn_off()` should be documented too or if just documenting `toggle()` is enough...
> > 
> > 
> > Please sorry, but i don't understand well right now. Can you send two variants of code for understanding your question? For example one where you documented only `toggle()` and another one where you documented all methods.
> > Please sorry i may be stupid in not understanding your question :) You can also do whatever you want.
> 
> My bad actually 😅,
> 
> By documenting i mean when making a pull request for the website, should i say the functions `turn_on()` and `turn_off` exist or not.
> 
> Sorry for not being easy to understand

Ah, thanks! Now i understand. Not, after merging this PR i (or maybe other person) will open PR with all changes in future version of Flet. You mustn't open PR. Actually, you can open PR and on this way you must document `turn_on()`, `turn_off()` and `toggle()` methods.